controllers/module_reconciler.go

```diff
@@ -340,21 +345,18 @@ func (mrh *moduleReconcilerHelper) handleSigning(ctx context.Context, mld *api.M
 }
 
 func (mrh *moduleReconcilerHelper) handleDriverContainer(ctx context.Context,
-	mld *api.ModuleLoaderData,
-	dsByKernelVersion map[string]*appsv1.DaemonSet) error {
+	mld *api.ModuleLoaderData) error {
+
+	hashValue, err := hashstructure.Hash(hashData{kernelVersion: mld.KernelVersion}, nil)
```

Comment:
If we just hash one string, I don't see the benefit of using hashstructure. It is probably much more expensive than [fnv](https://pkg.go.dev/hash/fnv).

Reply:
i want to use hashstructure, since later on we will add a module Version to the hash

Reply:
Turns out hashstructure does use fnv.